Components/Metrics/PointToSurfaceDistance/elxPointToSurfaceDistanceMetric.cxx

```diff
@@ -0,0 +1,17 @@
+/*======================================================================
+
+  This file is part of the elastix software.
+
+  Copyright (c) University Medical Center Utrecht. All rights reserved.
+  See src/CopyrightElastix.txt or http://elastix.isi.uu.nl/legal.php for
+  details.
+
+     This software is distributed WITHOUT ANY WARRANTY; without even
+     the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+     PURPOSE. See the above copyright notices for more information.
+
+======================================================================*/
```

Comment:
I think the license text is different now, please check the current version of other Elastix source files! Otherwise please ask Marius @mstaring

Reply:
> Can we copy from other files? I think should be the same ??

@gokhangg Yes, please, ensure that the copyright text is the same as in https://github.com/SuperElastix/elastix/blob/5.0.0/Core/Main/elastix.cxx

Reply:
Done

Reply:
Thanks!